lib/Common/Common/GetCurrentFrameId.h

```diff
@@ -21,8 +21,13 @@
 #define GET_CURRENT_FRAME_ID(f) \
     __asm { mov f, ebp }
 #elif defined(_M_X64)
+#ifdef _WIN32
 #define GET_CURRENT_FRAME_ID(f) \
     (f = _ReturnAddress())
+#else
+#define GET_CURRENT_FRAME_ID(f) \
+    __asm { mov f, rbp }
```

Comment:
> Isn't it interesting that on one platform we use frame pointer and returnadress on another ?

The reason is currently on x64 cross-platform is not on parity with Windows yet. We are effectively using calling convention and stack walker implementation the same as Windows x86. We may change these when working on JIT.